src/content/docs/en/guides/styling.mdx

```diff
@@ -429,9 +429,22 @@ import "../components/make-it-purple.css"
 
 ## CSS Integrations
 
-Astro comes with support for adding popular CSS libraries, tools and frameworks to your project like [Tailwind][tailwind] and more!
+Astro comes with support for adding popular CSS libraries, tools and frameworks to your project like Tailwind and more!
```

Comment:
I guess adding in the official Tailwind site here, couldn't hurt
```suggestion
Astro comes with support for adding popular CSS libraries, tools and frameworks to your project like [Tailwind](https://tailwindcss.com) and more!
```

Reply:
Agreed! Like it!